tests/unit/wordpress_mock.py

```diff
@@ -623,6 +623,11 @@ def _mock_a2disconf(self, cmd):
             pass
         return ExecProcessMock(return_code=0, stdout="", stderr="")
 
+    @_exec_handler.register(lambda cmd: cmd[:3] == ["wp", "core", "version"])
```

Comment:
I don't see any test that checks that the version is set correctly?

Reply:
Added a test to check the version set after the container ready event. Thank you :)